sdk/translation/azure-ai-translation-document/tests/asynctestcase.py

```diff
@@ -22,19 +22,20 @@ def generate_oauth_token(self):
                 os.getenv("TRANSLATION_CLIENT_SECRET"),
             )
 
-    async def _submit_and_validate_translation_job_async(self, async_client, translation_inputs, total_docs_count=None):
+    async def _begin_and_validate_translation_async(self, async_client, translation_inputs, total_docs_count=None, language=None):
```

Comment:
in which cases you will get here and not have documents to translate?

Reply:
and when is language none? sorry trying to understand the tests

Reply:
These are just options to pass if you want to validate certain properties. Let me look a little closer, it might need a bit of a refactor.